modin/core/io/column_stores/parquet_dispatcher.py

```diff
@@ -72,9 +71,12 @@ def _read(cls, path, engine, columns, **kwargs):
                     break
             partitioned_columns = list(partitioned_columns)
             if len(partitioned_columns):
-                ErrorMessage.default_to_pandas("Mixed Partitioning Columns in Parquet")
                 return cls.single_worker_read(
-                    path, engine=engine, columns=columns, **kwargs
+                    path,
+                    "Mixed partitioning columns in Parquet",
```

Comment:
Do you know what this error message is referring to? It's not clear what "mixed partitioning columns" means to me at least.

Reply:
I'm not sure (I just copied this from a warning in previous code).